### PR TITLE
fixes for minified version

### PR DESCRIPTION
**Pull request by darkdarkdragon**
test minified version in SauceLabs

### CODE REVIEW

**Comment by darkdarkdragon** [comment]
@clark800 please, re-run this build in circleci without cache

**Comment by darkdarkdragon** [comment]
@clark800 done

File: test/hacks/phantomhacks.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+// this one will be directly run in browser, so disable eslint
```
Comment (clark800) [inline test/hacks/phantomhacks.js]: We have to disable all eslint rules?

Comment (darkdarkdragon) [inline test/hacks/phantomhacks.js]: fixed

**Comment by clark800** [comment]
LGTM

**Pull request merged**